src/linked_list.py

```diff
@@ -0,0 +1,82 @@
+"""Creating a Linked List Data Structure.  CF 401 Python Week 2 assignment."""
+
+
+class Node(object):
+    """Create Node objects for use in a Linked List data structure."""
+    def __init__(self, value=None, nxt=None):
+        self.value = value
+        self.nxt = nxt
+
+
+class Linked_List(object):
+    """Create a Linked List Data Structure."""
+    def __init__(self, iterable=None):
+        """Intialize a Linked List Object."""
+        self.head = None
+        self._size = 0
+        if iterable and hasattr(iterable, "__iter__"):
```

Comment:
Note, and something I neglected during lecture, strings in Python 2 don't have the `__iter__` attribute. You may want to extend this with:
```python
if iterable and hasattr(iterable, "__iter__") or isinstance(iterable, str):
    # do stuff
```